Templates/make_jenkins_EC2-Agent-Windows-instance.tmplt.json

```diff
@@ -0,0 +1,1628 @@
+    "Mappings": {
+        "ShellCommandMap": {
+            "cmd": {
+                "command": "cmd.exe"
+            },
+            "powershell": {
+                "command": "powershell.exe -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Bypass "
+            }
+        }
+    },
+    "Metadata": {
+        "AWS::CloudFormation::Interface": {
+            "ParameterGroups": [
+                {
+                    "Label": {
+                        "default": "EC2 Instance Configuration"
+                    },
+                    "Parameters": [
+                        "AmiId",
+                        "AppVolumeMountPath",
+                        "AppVolumeSize",
+                        "AppVolumeType",
+                        "DnsSuffix",
+                        "EpelRepo",
+                        "InstanceType",
+                        "InstanceRole",
+                        "KeyPairName",
+                        "NoPublicIp",
+                        "NoReboot",
+                        "NoUpdates",
+                        "ProvisionUser",
+                        "PyStache",
+                        "PipRpm",
+                        "PipIndexFips",
+                        "SecurityGroupIds",
+                        "ServerHostname",
+                        "SubnetId"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Watchmaker Configuration"
+                    },
+                    "Parameters": [
+                        "PythonInstaller",
+                        "PypiIndexUrl",
+                        "WatchmakerBootstrapper",
+                        "WatchmakerConfig",
+                        "WatchmakerEnvironment",
+                        "WatchmakerOuPath",
+                        "WatchmakerComputerName",
+                        "WatchmakerAdminGroups",
+                        "WatchmakerAdminUsers"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Application Configuration"
+                    },
+                    "Parameters": [
+                        "AppScriptUrl",
+                        "AppScriptParams"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Application EBS Volume"
+                    },
+                    "Parameters": [
+                        "AppVolumeDevice",
+                        "AppVolumeSize",
+                        "AppVolumeType"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "Network Configuration"
+                    },
+                    "Parameters": [
+                        "PrivateIp",
+                        "SubnetId"
+                    ]
+                },
+                {
```

Comment:
Nuke this entire section (i.e., down to line 274): agent template isn't configuring an ELB